Index: oddprocessing.xsl
===================================================================
*** oddprocessing.xsl	(revision 9487)
--- oddprocessing.xsl	(working copy)
***************
*** 444 ****
--- 445,448 ----
+         <h2>Schema <xsl:value-of select="@ident"/>: Elements</h2>
+         <xsl:apply-templates mode="weave" select="tei:elementSpec">
+           <xsl:sort select="@ident"/>
+         </xsl:apply-templates>
***************
*** 463,466 ****
-         <h2>Schema <xsl:value-of select="@ident"/>: Elements</h2>
-         <xsl:apply-templates mode="weave" select="tei:elementSpec">
-           <xsl:sort select="@ident"/>
-         </xsl:apply-templates>
--- 466 ----
